startlxqtwayland.in

```diff
@@ -70,7 +70,7 @@ if [ ! -d "$XDG_CONFIG_HOME/lxqt/wayland" ]; then
 fi
 
 if grep -q "compositor" "$XDG_CONFIG_HOME/lxqt/session.conf"; then
```

Comment:
The whole `if ... fi` block can be replaced with following assignment:
```sh
COMPOSITOR="$(sed -nre '/^compositor\s*=/ { s/^[^=]+=\s*//; s@^.*/([^/]*)$@\1@; p; }'  ~/.config/lxqt/session.conf)"
```
Just mu humble 2c

Reply:
Looks like a cool artwork, thanks! IMO better leave it more readable.